src/test/java/com/crud/tasks/mapper/TrelloMapperTestSuite.java

```diff
@@ -24,49 +24,53 @@
-        TrelloBoard trelloBoard = trelloMapper.mapToBoards(trelloBoardDtoList).get(0);
+        List<TrelloBoard> resultTrelloBoard = trelloMapper.mapToBoards(trelloBoardDtoList);
         //Then
-        assertEquals(trelloBoardDtoList.get(0).getId(), trelloBoard.getId());
-        assertEquals(trelloBoardDtoList.get(0).getName(), trelloBoard.getName());
-        assertEquals(trelloBoardDtoList.get(0).getLists().size(),trelloBoard.getLists().size());
+        assertEquals(trelloBoardDtoList.get(0).getId(), resultTrelloBoard.get(0).getId());
+        assertEquals(trelloBoardDtoList.get(0).getName(), resultTrelloBoard.get(0).getName());
+        assertEquals(trelloBoardDtoList.get(0).getLists().size(),resultTrelloBoard.get(0).getLists().size());
+        assertEquals(trelloBoardDtoList.size(), resultTrelloBoard.size());
     }
 
     @Test
     public void testMapToBoardsDto() {
         //Given
-        List<TrelloBoard> trelloBoardList = List.of(new TrelloBoard("test_id","test_board",new ArrayList<>()));
+        List<TrelloBoard> trelloBoardList = List.of(new TrelloBoard("test_id", "test_board", new ArrayList<>()));
         //When
-        TrelloBoardDto resultTrelloBoardDto = trelloMapper.mapToBoardsDto(trelloBoardList).get(0);
+        List<TrelloBoardDto> resultTrelloBoardDto = trelloMapper.mapToBoardsDto(trelloBoardList);
         //Then
-        assertEquals(trelloBoardList.get(0).getId(), resultTrelloBoardDto.getId());
-        assertEquals(trelloBoardList.get(0).getName(), resultTrelloBoardDto.getName());
-        assertEquals(trelloBoardList.get(0).getLists().size(), resultTrelloBoardDto.getLists().size());
+        assertEquals(trelloBoardList.get(0).getId(), resultTrelloBoardDto.get(0).getId());
+        assertEquals(trelloBoardList.get(0).getName(), resultTrelloBoardDto.get(0).getName());
+        assertEquals(trelloBoardList.get(0).getLists().size(), resultTrelloBoardDto.get(0).getLists().size());
+        assertEquals(trelloBoardList.size(), resultTrelloBoardDto.size());
     }
 
     @Test
     public void testMapToList() {
         //Given
         List<TrelloListDto> trelloListsDto = List.of(new TrelloListDto("1", "list_name", false));
         //When
-        TrelloList resultTrelloList = trelloMapper.mapToList(trelloListsDto).get(0);
+        List<TrelloList> resultTrelloList = trelloMapper.mapToList(trelloListsDto);
         //Then
-        assertEquals(trelloListsDto.get(0).getId(), resultTrelloList.getId());
-        assertEquals(trelloListsDto.get(0).getName(), resultTrelloList.getName());
-        assertEquals(trelloListsDto.get(0).isClosed(), resultTrelloList.isClosed());
+        assertEquals(trelloListsDto.get(0).getId(), resultTrelloList.get(0).getId());
+        assertEquals(trelloListsDto.get(0).getName(), resultTrelloList.get(0).getName());
+        assertEquals(trelloListsDto.get(0).isClosed(), resultTrelloList.get(0).isClosed());
+        assertEquals(trelloListsDto.size(), resultTrelloList.size());
```

Comment:
Kosmetyczna uwaga: lepiej `size` sprawdzać jako pierwszą asercję. Chodzi o to, żeby w przypadku błędu (np. mapper zwraca pustą listę), test sfailował na asercji, a nie wyjątkiem `IndexOutOfBounsException` po wywołaniu `get(0)`.